exercises/G-numbers/exercise.js

```diff
@@ -1 +1,11 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15
+var numberOfMentors = 8
+var totalOf = 15 + 8;
```

Comment:
Hey Rafael :)
Here we are repeating ourselves by using the same numbers we used in the variables `numberOfStudents` and `numberOfMentors`.
Is there a way we can avoid using these numbers again? (Hint: you solved it in the exercise **I-floats**)